index.html

```diff
@@ -0,0 +1,187 @@
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  </div>
+      
+  
+    <div class="bg-image">
+      <div class="main-section">
+        <div class="heading">
+          <p class="header-paragraph">"Computer Workshop"</p>
+          <h1>MAINTENANCE AND SOFTWARE INSTALLATION</h1>
+        </div>
+    
+        <div class="header-text">
+          <p class="pad-class grey text-bg">We provide basic computer maintenance services and we also install softwares of all kinds
+            both system and application softwares
+          </p>
+        </div>
+    
+        <div class="timing">
+          <h3 class="grey">2023.01.02(MON) ~ 15(TUES)</h3>
+          <p class="grey">Curated by Higher Grounds Technological Center</p>
+        </div>
+  
+      </div>
+    </div>    
+  
+    </div>
+
+    <div class="enclose"></div>
+
+    <div class="program">
+      <div class="section-text">
+        <h4>What we do</h4>
+        <div class="program-line"></div>
+      </div>
+
+      <div class="card-container">
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            computer
+          </span>
+            <p class="card-margin technology">Repairs</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            tools_installation_kit
+          </span>
+            <p class="card-margin technology">Install</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            quiz
+            </span>
+            <p class="card-margin technology">Consult</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            point_of_sale
+            </span>
+            <p class="card-margin technology">Sales</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+        <div class="card-content">
+          <span class="material-symbols-outlined card-margin">
+            cast_for_education
+            </span>
+            <p class="card-margin technology">Training</p>
+          <p class="content-paragraph">Computer repairs and maintenance for better system performance.
+          </p>
+        </div>
+  
+      </div>
+      <a href="" class="more-link">See The Whole Services We Offer</a>
+
+
+
+      <button class="btn">Sign up for business</button>
+
```

Comment:
- [ ] As requested by the previous please make sure you stick to the design as much as possible, I suggest centering this button as shown below :point_down: 
- Your design:
![yjoinBTN](https://user-images.githubusercontent.com/97834160/210792727-51e276c3-71be-4647-89e4-2a38abef216c.png)
 - Behance:
 
![FjoibBTN](https://user-images.githubusercontent.com/97834160/210792865-78391918-69f6-49a6-b1c3-6f0dd2ea0223.png)